handsontable/src/plugins/filters/filters.js

```diff
@@ -338,6 +353,44 @@ export class Filters extends BasePlugin {
    * filtersPlugin.addCondition(1, 'not_contains', ['ing'], 'disjunction');
    * filtersPlugin.filter();
    * ```
+   * :::
+   *
+   * ::: only-for react
+   * ```jsx
+   * const hotRef = useRef();
+   *
+   * ...
+   *
+   * <HotTable
+   *   ref={hotRef}
+   *   data={getData()}
+   *   filters={true}
+   * />
+   *
+   * // access to filters plugin instance
+   * const filtersPlugin = hotRef.getPlugin('filters');
```

Comment:
Same as above - you'll need to call the API methods from the `.current.hotInstance` property of the referenced component.

Reply:
Changed within: https://github.com/handsontable/handsontable/pull/9791/commits/8464a8efd8e87971f4d593025622e98b0e8b20ae